src/main/docker/gitlab/gitlab-local-setup.sh

```diff
@@ -0,0 +1,55 @@
+
+#!/bin/bash
+
+# Enter the personal access token of the admin user
+ADMIN_PERSONAL_ACCESS_TOKEN="your.gitlab.admin.access.token"
+ADMIN_USER_ID="1"
+
+# Enter the url of the Gitlab instance.
+GITLAB__API_URL="http://localhost/api/v4/"
```

Comment:
I would separate the values the user has to set and the values 'calculated'

So maybe just arrange ADMIN_PERSONAL_ACCESS_TOKEN and GITLAB__API_URL to the top. I don't think the other values have to be changed by default. (Especially, I think you have no need to set ADMIN_USER_ID AUTHORIZATION_BEARER_HEADER manually)